front/components/assistant/AssistantActions.tsx

```diff
@@ -87,7 +88,7 @@ export function RemoveAssistantFromListDialog({
   onRemove,
 }: {
   owner: WorkspaceType;
-  agentConfiguration: AgentConfigurationType;
+  agentConfiguration: AgentConfigurationListViewType;
```

Comment:
I thiiiink we want to find a standard naming to use for variable names. Not super fan of `detailConfig` as an example here.

How about:
`lightAgentConfiguration` and `agentConfiguration` ?

Open to other proposition.

Also having second thoughts on "ListView". `Light` seems more generic?

Reply:
Light seems better indeed. Updating